apps/projects/app/components/Content/IssueDetail/DetailsCard.js

```diff
@@ -173,6 +173,7 @@ const DetailsCard = ({ issue }) => {
               style={{ marginRight: '5px', marginTop: '20px' }}
               background={'#' + label.node.color + '99'}
               color={`${theme.content}`}
+              uppercase={false}
```

Comment:
We can probably just omit the prop and it will be `false`, not a big deal, just in case you didn't know that @rkzel 
```suggestion
```